api-report/container-definitions.api.md

```diff
@@ -78,8 +78,7 @@ export interface IAudience extends EventEmitter {
 
 // @public
 export interface IAudienceOwner extends IAudience {
-    addMember(clientId: string, details: IClient): any;
-    clear(): any;
+    addMember(clientId: string, details: IClient): void;
```

Comment:
Not sure why the type validation tests didn't catch this... These are breaking changes which require the `package.json` declarations + BREAKING.md notes. Same for the `ProtocolHandlerBuilder` change

Reply:
Yep, I saw similar issue somewhere else - see here: https://github.com/microsoft/FluidFramework/pull/12058#discussion_r979028935

Reply:
Opened https://dev.azure.com/fluidframework/internal/_workitems/edit/2026 to track it